src/github.rs

```diff
@@ -13,45 +13,177 @@ use crate::{database::Database, github::{
     pull_requests::PullRequestsRepositoryPullRequestsNodesReviewRequestsNodesRequestedReviewer::User,
 }, settings::Repository as RepoInfo};
 
+pub(crate) use pull_requests::{PullRequestState, PullRequestReviewState};
+
 const GITHUB_FETCH_SIZE: i64 = 10;
 const GITHUB_URL: &str = "https://api.github.com/graphql";
 const APP_USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION"),);
 const INIT_TIME: &str = "1992-06-05T00:00:00Z";
 
-type DateTime = String;
+pub(crate) type DateTime = Timestamp;
 
+#[allow(clippy::upper_case_acronyms)]
+type URI = String;
 #[derive(GraphQLQuery)]
 #[graphql(
     schema_path = "src/github/schema.graphql",
     query_path = "src/github/issues.graphql",
-    response_derives = "Debug"
+    response_derives = "Debug",
+    scalar = "DateTime = DateTime"
 )]
 struct Issues;
 
 #[derive(GraphQLQuery)]
 #[graphql(
     schema_path = "src/github/schema.graphql",
-    query_path = "src/github/pull_requests.graphql"
+    query_path = "src/github/pull_requests.graphql",
+    response_derives = "Debug, Clone, PartialEq, Eq",
+    scalar = "DateTime = DateTime",
+    scalar = "URI = URI"
 )]
 struct PullRequests;
 
 #[derive(Debug)]
-pub(super) struct GitHubIssue {
-    pub(super) number: i64,
-    pub(super) title: String,
-    pub(super) author: String,
-    pub(super) closed_at: Option<DateTime>,
+pub struct GitHubIssue {
+    pub number: i64,
+    pub title: String,
+    pub author: String,
```

Comment:
If `pub` is not strictly required for this code and the code below, please replace it with `pub(crate)`.